src/backend/gporca/libnaucrates/src/md/CMDTypeGenericGPDB.cpp

```diff
@@ -518,6 +520,32 @@ CMDTypeGenericGPDB::IsTimeRelatedType(const IMDId *mdid)
 		   mdid->Equals(&CMDIdGPDB::m_mdid_time_interval);
 }
 
+//---------------------------------------------------------------------------
+//	@function:
+//		CMDTypeGenericGPDB::IsTimeRelatedTypeMappableToDouble
+//
+//	@doc:
+//		is this a time-related type that is mappable to double
+//---------------------------------------------------------------------------
+inline BOOL
```

Comment:
Signatures of these functions don't match with definitions in headers. Do we really need to explicitly mark these functions as inline?

Reply:
Thanks. I have moved the definition of helper function into header.